.github/workflows/build.yml

```
@@ -330,3 +331,55 @@ jobs:
           name: msys2-${{matrix.boards}}-builds
           path: buildartifacts/
         continue-on-error: true
+
+  # Select the msvc Builds based on PR Arch Label
+  msvc-Arch:
+    uses: apache/nuttx/.github/workflows/arch.yml@master
```

```suggestion
    uses: apache/nuttx-apps/.github/workflows/arch.yml@master
```
Sorry this should point to nuttx-apps, so that changes to the nuttx workflow won't break nuttx-apps. Thanks :-)

---

@lupyuen done